tests/e2e/tests/commands/get/get.ts

```diff
@@ -4,6 +4,7 @@ import {expectToFail} from '../../../utils/utils';
 export default function() {
   return Promise.resolve()
     .then(() => expectToFail(() => ng('get', 'apps.zzz.prefix')))
+    .then(() => expectToFail(() => ng('get', undefined)))
```

Comment:
This should be just `ng('get')` as `ng get undefined` seems to be working fine already as it is converted to string key.